docs/CaliptraIntegrationSpecification.md

```diff
@@ -725,6 +725,145 @@ The following code snippet and schematic diagram illustrate JTAG originating CDC
+        - key_vault1.kv_reg1.s_cpuif_req
+        - pcr_vault1.pv_reg1.s_cpuif_req
+        - data_vault1.dv_reg1.s_cpuif_req
+        - sha512.sha512_inst.i_sha512_reg.s_cpuif_req
+        - soc_ifc_top1.i_ahb_slv_sif_soc_ifc.dv
+        - sha256.sha256_inst.i_sha256_reg.s_cpuif_req
+        - csrng.u_reg.u_ahb_slv_sif.dv
+        - entropy_src.u_reg.u_ahb_slv_sif.dv
+6. Constraint the RDC false paths as per *Table 21*.
+
+
+## RDC Violations and Waivers
+
+Considering the given constraints, we've identified three sets of crossings that are flagged as RDC violations. All of them can be waived as explained in *Table 22*. It's worth noting that the report may differ across EDA tools due to variations in structural analysis, which can be influenced by a range of settings.
+
+*Table 22: Reset Domain Crossing Violations*
+
+| Sl No | Launch Reset | Launch Flop | Capture Reset | Capture Flop |
+| ------| ------------ | ----------- | ------------- | ------------ |
+| 1 | cptra_uc_rst_b | rvtop.veer.dec.tlu.exthaltff.genblock.dff.dout[7] | cptra_noncore_rst_b | cg.user_soc_ifc_icg.clk_slcg_0_generated_icg.p_clkgate.vudpi0.q           |
+| 2 | cptra_uc_rst_b | rvtop.veer.dec.tlu.exthaltff.genblock.dff.dout[7] | cptra_noncore_rst_b | cg.user_icg.clk_slcg_0_generated_icg.p_clkgate.vudpi0.q                   |
+| 3 | cptra_uc_rst_b | rvtop.veer.Gen_AXI_To_AHB.lsu_axi4_to_ahb*        | cptra_noncore_rst_b | entropy_src.u_entropy_src_core.u_caliptra_prim_packer_fifo_bypass.data_q* |
+| 4 | cptra_uc_rst_b | rvtop.veer.Gen_AXI_To_AHB.lsu_axi4_to_ahb*        | cptra_noncore_rst_b | entropy_src.u_entropy_src_core.u_caliptra_prim_packer_fifo_precon.data_q* |
+
+<br>
+
+**For violations in Sl No 1 and 2**, below in the schematic for the crossing. 
+
+![](./images/halt_status_rdc.png)
+
+We can waive them because if the cpu is halted, there is no way to trigger fw update reset as it is initiated by the uc itself. So we can ensure that on *cptra_uc_rst_b*, *cpu_halt_status* will be at the reset value already, which will ensure that there are no glitches on the output clock of CG.
+
+**For violations in Sl No 3 and 4**,
+
+![](./images/entropy_RDC.png)
+
+If we see the above schematic, we have the RDC crossing violation at data_q flop (on the right-hand side). We are explaining the analysis for prim_packer_fifo_bypass here. Same holds true for prim_packer_fifo_recon.
+
+The fifo load and wr_data equations are as follows:-
+
+<span style="font-family: Consolas;">
+  assign pfifo_bypass_push = !es_bypass_mode ? 1'b0 :
+                             fw_ov_mode_entropy_insert ? fw_ov_fifo_wr_pulse :
+                             pfifo_postht_not_empty;
+  assign pfifo_bypass_wdata = fw_ov_mode_entropy_insert ? fw_ov_wr_data :
+                              pfifo_postht_rdata;
+</span>
+
+<br>
+Below are the various scenarios that can happen.
+
+| #Case | es_bypass_mode | fw_ov_mode_entropy_insert | pfifo_bypass_push | pfifo_bypass_wdata | Comment |
```

Comment:
Should this also be a numbered table?
We've also been captioning and numbering Figures in the document  - could you make that update as well for new images?

Reply:
thanks, done